scripts/feature_engineering.ipynb

```diff
@@ -0,0 +1,312 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "**Description** \n",
+    "\n",
+    "This Jupyter Notebook contains code for feature engineering and then setting up different model experiments. \n",
+    "\n",
+    "**Author**: Phoebe Ly, Vivek Sakhrani, Isabella Smythe\n",
+    "\n",
+    "**Last Updated:** 08-MAR-2025\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import geopandas as gpd\n",
+    "import rasterio\n",
+    "from rasterstats import zonal_stats\n",
+    "from rasterio.features import geometry_window\n",
+    "from rasterio.sample import sample_gen\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from sklearn.linear_model import LinearRegression\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.ensemble import RandomForestRegressor\n",
+    "from sklearn.metrics import mean_squared_error, r2_score\n",
+    "from sklearn.preprocessing import StandardScaler\n",
+    "from sklearn.multioutput import MultiOutputRegressor\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create buffer around HH points\n",
+    "peoplesun_hhapps_anon = pd.read_csv('gs://ads-lacuna-ai4ea/data/dataverse/peoplesun_hhapps_anon.csv') #Make sure data source\n",
```

Comment:
I don't have access to `gs://ads-lacuna-ai4ea/data/dataverse/peoplesun_hhapps_anon.csv`

I am not sure how I am supposed to use this nb if I cannot run it pass the second cell

Reply:
@vivek-atlasai